src/Commands/BuildCommands.php

```diff
@@ -141,7 +140,24 @@ public function buildBranch($branch, $options = ['clean' => TRUE]) {
     }
 
     // Handle .gitignore.
-    if ($symfony_fs->exists('.build-gitignore')) {
+    $finder = new Finder();
+    $finder
+      ->name('.gitignore-build')
+      ->ignoreDotFiles(FALSE)
+      ->in(getcwd());
+
+    if ($finder->count() != 0) {
+      $this->say('Found .gitignore-build - applying it.');
+      foreach ($finder as $file) {
+        $filePathname = $file->getPathname();
+        $gitignorePathname = $file->getPath() . '/.gitignore';
+        $this->taskExec("$filePathname >> $gitignorePathname");
```

Comment:
missing `cat` in the command, since with current state this is not attaching anything to .gitignore